### ISSUE TITLE
Alt text not visible in images

### ISSUE THREAD

**Issue by bonzini** [pretalx/pretalx-pages#4]
The following Markdown:

```markdown
![Red Hat](/static/images/red-hat.svg)
```

results in the following HTML:

```html
<img src="/static/images/red-hat.svg">
```

The text of the link should be reproduced as a title or alt attribute.
